intranet/templates/eighth/take_attendance.html

```diff
@@ -432,6 +432,13 @@ <h3>Passes</h3>
                             {% endfor %}
                         </tbody>
                         <tfoot class="print-hide">
+                            <tr>
+                                <td colspan="5">
+                                    <br>
+                                    <b>Take Attendance with Google Meet CSV Report:</b>
+                                    <input type="file" name="attendance" accept=".csv">
+                                </td>
+                            </tr>
```

Comment:
This, for some reason, intrudes on the "passes" part:

![image](https://user-images.githubusercontent.com/3579871/107082976-f19f5700-67c2-11eb-9f81-2e0e6e9bf454.png)